test/Integration/attack.spec.ts

```diff
@@ -38,9 +39,9 @@ describe('Test Attack service requests', () => {
 		return request(server)
 			.post('/attack/Fire')
 			.query({ ...params })
-			.then(res => {
-				expect(res.statusCode).toBe(200);
-				expect(res.headers['content-type']).toBe('application/json; charset=utf-8');
+			.then((res: supertest.Response) => {
```

Comment:
We have CLI which generates integration spec for generated services, can you also update template file to have these updates?

`project-cli/helper_scripts/Templates/Tests/IntegrationTest.mustache`